patch-files/pydeceive/CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+
+### 2. Clone it :busts_in_silhouette:
+
+You need to clone (download) it to local machine using
+
+```sh
+git clone https://github.com/Your_Username/PyDeceive.git
+```
+where 'username' is your username.
+
+> You’ll now have a local copy of your version of the original FOSSASIA repository.
+Once you have cloned the ` PyDeceive ` repository in GitHub, move to that folder first using change directory command.
+
+```sh
+# This will change directory to a folder PyDeceive
+cd PyDeceive
+```
+
+Move to this folder for all other commands.
+
+### 3. Set it up :arrow_up:
+
+Run the following commands to see that *your local copy* has a reference to *your forked remote repository* in GitHub :octocat:
+
+```sh
+git remote -v
+origin  https://github.com/Your_Username/PyDeceive.git (fetch)
+origin  https://github.com/Your_Username/PyDeceive.git (push)
+```
+
+Now, add a reference to the original [PyDeceive](https://github.com/smaranjitghose/PyDeceive) repository using
+
+```sh
+git remote add upstream https://github.com/smaranjitghose/PyDeceive.git
+```
+
+> This adds a new remote named ***upstream***.
+See the changes using
+
+```sh
+git remote -v
+origin    https://github.com/Your_Username/PyDeceive.git (fetch)
+origin    https://github.com/Your_Username/PyDeceive.git (push)
+upstream  https://github.com/smaranjitghose/PyDeceive.git (fetch)
+upstream  https://github.com/smaranjitghose/PyDeceive.git (push)
+```
+
+### 4. Sync it :recycle:
+
+Always keep your local copy of repository updated with the original repository.
+Before making any changes and/or in an appropriate interval, run the following commands *carefully* to update your local repository.
+
+```sh
+# Fetch all remote repositories and delete any deleted remote branches
+git fetch --all --prune
+# Switch to `master` branch
+git checkout master
+# Reset local `master` branch to match `upstream` repository's `master` branch
+git reset --hard upstream/master
+# Push changes to your forked `PyDeceive` repo
+git push origin master
+```
+
+### 5. Ready Steady Go... :turtle: :rabbit2:
+
+Once you have completed these steps, you are ready to start contributing by checking our `Help Wanted` Issues and creating [pull requests](https://github.com/smaranjitghose/PyDeceive/pulls).
+
+### 6. Create a new branch :bangbang:
+
+Whenever you are going to make contribution. Please create separate branch using command and keep your `master` branch clean (i.e. synced with remote branch).
+
+```sh
+# It will create a new branch with name Branch_Name and will switch to that branch.
+git checkout -b Branch_Name
+```
+
+Create a separate branch for contribution and try to use same name of branch as of folder.
+
+To switch to desired branch
+
+```sh
+# To switch from one folder to other
+git checkout Branch_Name
+```
+
+To add the changes to the branch. Use
+
+```sh
+# To add all files to branch Branch_Name
+git add .
```

Comment:
hey bro..
here you can do git add path-to-your-file. because i think thats more appropiate.

Reply:
Any specific reason..?